source/docker/index.rst

```diff
@@ -1,5 +1,8 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
+.. meta::
+  :description: Learn more about the process of installing and configuring the Wazuh deployment on Docker in this section of our documentation. 
```

Comment:
I can still see the old meta description under the title of this section (lines 8/11) and it needs to be removed. 
If you've fixed it later on, dismiss this comment.